buf.gen.yaml

```diff
@@ -1,18 +1,18 @@
 ---
 version: "v1"
 plugins:
-  - remote: "buf.build/library/plugins/go:v1.27.1-1"
+  - plugin: "buf.build/protocolbuffers/go:v1.30.0"
     out: "proto"
     opt: "paths=source_relative"
-  - remote: "buf.build/library/plugins/go-grpc:v1.1.0-2"
+  - plugin: "buf.build/grpc/go:v1.3.0"
     out: "proto"
     opt: "paths=source_relative"
-  - remote: buf.build/odpf/plugins/validate
```

Comment:
I presume this was meant to be https://github.com/bufbuild/protoc-gen-validate, we've since broken up this plugin into {go,cpp,java} and no longer need to add the `lang=go` plugin option.